.github/workflows/test.yml

```diff
@@ -83,15 +80,10 @@ jobs:
         ]
 
         include:
-          - name: "windows-py38"
-            python: "3.8"
-            os: windows-latest
-            tox_env: "py38-unittestextras"
```

Comment:
i believe this one need a python 3.9 variant as well,

Reply:
I've been meaning to ask why this only uses inclusions? Usually, combining them with builtin matrix generation works better..

Reply:
im not sure what you mean, i believe we had a number of special/slow envs that are only used for very specific tests (freeze, pypy, pluggy) and as far as i know this was the best working solution known at the time - if there is something better, i'd love to see that

as far as i understand/remember the builtin matrix would have generated a variant product, when we want a subset

Reply:
Should we name it "windows-py38-unittestextras" then ? I initially put it back then removed because the name was not "special" (the toxenv name is though).

Reply:
The unittestextra tox factor adds twisted and asynctest as dependencies

Reply:
Fixed in [284d04f](https://github.com/pytest-dev/pytest/pull/12875/commits/284d04fe041473dab2d0037be7360decd8ef4444) as far as I understand

Reply:
@RonnyPfannschmidt I'll look into the possibility to use the matrix+exclude separately at some point.